test/int/wizards/breakpoints/implementation/breakpointsAssertions.ts

```diff
@@ -57,17 +94,28 @@ export class BreakpointsAssertions {
     public async assertIsHitThenResume(breakpoint: BreakpointWizard, verifications: IVerifications): Promise<void> {
         await this._breakpointsWizard.waitUntilPaused(breakpoint);
 
-        const stackTrace = await this.stackTrace();
-        const topFrame = stackTrace[0];
+        let stackFrameFormat: DebugProtocol.StackFrameFormat = this._defaultStackFrameFormat;
+        let stackTraceVerifier: IStackTraceVerifier | undefined = undefined;
+
+        if (verifications.stackTrace && verifications.stackTraceVerifier) {
```

Comment:
What do you think, instead of passing the verifications.stackTraceVerifier, checking whether verifications.stackTrace is a string or an object, and then automatically using the corresponding verifier?

Reply:
I wish we were able to check the interfaces directly, instead of checking string vs object. Otherwise I think it's a good idea. I made the change, which I think is cleaner now.